### PR TITLE
Autoplot - Enable mssautoplot for side and linearviews

### PR DESCRIPTION
**Pull request by Preetam-Das26**
**Purpose of PR?**:
Enable mssautoplot for side and linearviews
Fixes #

**Does this PR introduce a breaking change?**

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Does this PR results in some Documentation changes?**
_If yes, include the list of Documentation changes_

**Checklist:**
- [ ] Bug fix. Fixes #<issue number>
- [ ] New feature (Non-API breaking changes that adds functionality)
- [ ] PR Title follows the convention of  `<type>: <subject>`
- [ ] Commit has unit tests

### CODE REVIEW

File: mslib/utils/mssautoplot.py [hunk 1]
```diff
@@ -421,12 +433,16 @@ def draw(self, flight, section, vertical, filename, init_time, time, url, layer,
 
 class LinearViewPlotting(Plotting):
     # ToDo Implement access of MSColab
-    def __init__(self, cpath):
+    def __init__(self, cpath, msc_url, msc_auth_password, msc_username, msc_password):
         super(LinearViewPlotting, self).__init__(cpath)
```
Comment (ReimarBauer) [inline mslib/utils/mssautoplot.py]: comparing with class TopViewPlotting(Plotting):

We have there `super(TopViewPlotting, self).__init__(cpath, msc_url, msc_auth_password, msc_username, msc_password)` too.

Same question also for `super(SideViewPlotting, self).__init__(cpath)`

Is that required? If not please look if it is in TopViewPlotting required too or could be changed to a similiar.

Comment (Preetam-Das26) [reply mslib/utils/mssautoplot.py]: Yes that is required. Changes Done.

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
In different to topview and linearview the sideview does not use the mscolab server, it looks for a filename new1 in that example. Please improve this.

not_using_mscolab


UPDATE: I just recognized that in mssautoplot, in draw "linear" is not yet implemented. Work to implement linear should go in a different PR. Focus on get sideview with mscolab working as topview does.

**Review by ReimarBauer — APPROVED** [review]
will be continued as discussed on slack in a different PR

**Pull request merged**